conda-environment.yaml

```diff
@@ -8,3 +8,4 @@ dependencies:
   - pip=23.3.1
   - pip:
       - polygon3==3.0.9.1
+      - shapely==2.0.3
```

Comment:
Shouldn't this requirement also be added to `requirements.txt` for pip-based installs?